test/general-tests/devnet-restart/check.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+npx hardhat starknet-compile
```

Comment:
If you don't need all contracts from the repo, you can compile only the ones you need.

Reply:
[ea3c5bf](https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/89/commits/ea3c5bf494389944fb7c1db77fed48e343fe8fa0)